vm/systemSmartContracts/staking.go

```diff
@@ -1345,6 +1347,30 @@ func (r *stakingSC) setOwner(args *vmcommon.ContractCallInput) vmcommon.ReturnCo
 	return vmcommon.Ok
 }
 
+func (r *stakingSC) getOwner(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if !r.flagSetOwner.IsSet() {
+		r.eei.AddReturnMessage("invalid method to call")
+		return vmcommon.UserError
+	}
+	if !bytes.Equal(args.CallerAddr, r.stakeAccessAddr) {
+		r.eei.AddReturnMessage("this is only a view function")
+		return vmcommon.UserError
+	}
+	if len(args.Arguments) < 1 {
+		r.eei.AddReturnMessage(fmt.Sprintf("invalid number of arguments: expected min %d, got %d", 1, len(args.Arguments)))
+		return vmcommon.UserError
+	}
+
```

Comment:
We should not check also here if args.CallValue.Cmp(zero) != 0 and r.eei.UseGas(r.gasCost.MetaChainSystemSCsCost.Get)?

Reply:
not required. The function can be called only by the auction address, as seen on L1355

Reply:
ok